packages/grafana-ui/src/components/Button/Button.story.tsx

```diff
@@ -88,4 +104,8 @@ export const Variants: ComponentStory<typeof Button> = () => {
   );
 };
 
-export default meta;
+export const Default = ({ ...args }: ButtonProps) => <Button {...args} />;
+
+Default.args = {
+  children: 'Example button',
+};
```

Comment:
We should add the other defaults here for the controls

```suggestion
Default.args = {
  children: 'Example button',
  size: 'md',
  variant: 'primary',
  fill: 'solid',
};
```

Reply:
Done